newIDE/app/src/Utils/Analytics/EventSender.js

```diff
@@ -146,6 +146,18 @@ export const sendExportLaunched = (exportKind: string) => {
   });
 };
 
+export const sendExampleDetailsOpened = ({
+  id,
+  name,
+}: {|
```

Comment:
`sendNewGameCreated` send the project url. It won't be easy to follow.
The url is not known until the full example information are get. Do you think the id could be used instead?

Reply:
Fair enough, then in this case I would recommend to send more info: `sendNewGameCreated({ templateName: example.projectFileUrl, exampleId: example.id, exampleSlug: example.slug })`

So we still have the old information ("templateName"... not super adapted but well that's what we have already) and the id and slug to do some stats (id because it's unique, slug because it's more readable and also unique!).